jax/_src/numpy/polynomial.py

```diff
@@ -231,9 +231,66 @@ def polyval(p: Array, x: Array, *, unroll: int = 16) -> Array:
   y, _ = lax.scan(lambda y, p: (y * x + p, None), y, p, unroll=unroll)
   return y
 
-@implements(np.polyadd)
+
 @jit
 def polyadd(a1: Array, a2: Array) -> Array:
+  r"""Returns the sum of the two polynomials.
+
+  JAX implementation of :func:`numpy.polyadd`.
+
+  Args:
+    a1: Array of polynomial coefficients.
+    a2: Array of polynomial coefficients.
+
+  Returns:
+    An array containing the coefficients of the sum of input polynomials.
+
+  Note:
+
+    :func:`jax.numpy.polyadd` only accepts arrays as input unlike
+    :func:`numpy.polyadd` which accepts scalar inputs as well.
```

Comment:
There is no reason for this API difference to exist – should we change it?

Reply:
Yes, it would be better if we can make JAX follow NumPy's behavior here. Do you want me remove this 'Note'?

Reply:
Well, let's keep the note for now. We can remove it if we change the API